samples/csharp/getting-started/Ranking_PersonalizedSort/README.md

```diff
@@ -0,0 +1,215 @@
+    .Select(s => s.Name)
+    .Where(c =>
+        c == nameof(HotelData.Price_USD) ||
+        c == nameof(HotelData.Promotion_Flag) ||
+        c == nameof(HotelData.Prop_Id) ||
+        c == nameof(HotelData.Prop_Brand) ||
+        c == nameof(HotelData.Prop_Review_Score))
+    .ToArray();
+
+// Set trainer options.  
+LightGbmRankingTrainer.Options options = new LightGbmRankingTrainer.Options();
+options.CustomGains = new int[] { 0, 1, 5 };
+options.RowGroupColumnName = nameof(HotelData.GroupId);
+options.LabelColumnName = nameof(HotelData.Label);
+options.FeatureColumnName = FeaturesVectorName;
+
+// Create an Estimator and transform the data:
+// 1. Concatenate the feature columns into a single Features vector.
+// 2. Create a key type for the label input data by using the value to key transform.
+// 3. Create a key type for the group input data by using a hash transform.
+IEstimator<ITransformer> dataPipeline = mlContext.Transforms.Concatenate(FeaturesVectorName, featureCols)
+    .Append(mlContext.Transforms.Conversion.MapValueToKey(nameof(HotelData.Label)))
+    .Append(mlContext.Transforms.Conversion.Hash(nameof(HotelData.GroupId), nameof(HotelData.GroupId), numberOfBits: 20));
+
+// Set the LightGbm Lambdarank trainer.
+IEstimator<ITransformer> trainer = mlContext.Ranking.Trainers.LightGbm(options);
+IEstimator<ITransformer> trainerPipeline = dataPipeline.Append(trainer);
+
+// Training the model is a process of running the chosen algorithm on the given data. To perform training you need to call the Fit() method.
+ITransformer model = trainerPipeline.Fit(trainData);
+
+// Save the model
+ mlContext.Model.Save(model, trainData.Schema, modelPath);
+`````
+
+### 4. Test and Evaluate Model
+We need this step to conclude how accurate our model is. To do so, the model from the previous step is run against another dataset that was not used in training (e.g. the test dataset).  
+
+`Evaluate()` compares the predicted values for the test dataset and produces various metrics, such as accuracy, you can explore.  Specifically, we can gauge the accuracy of our model using Discounted Cumulative Gain (DCG) and Normalized Discounted Cumulative Gain (NDCG) which are included in the `RankingMetrics` returned by `Evaluate()`. 
+
+When evaluating the `RankingMetrics` for this sample's model, you'll notice that the following metrics are reported for DCG and NDCG (the values that you see when running the sample will be similar to these):
+* DCG - @1:1.0191, @2:1.5128, @3:1.8371, @4:2.0922, @5:2.2982, @6:2.4641, @7:2.6051, @8:2.7240, @9:2.8234, @10:2.9133
+
+* NDCG - @1:0.1184, @2:0.1719, @3:0.2082, @4:0.2372, @5:0.2608, @6:0.2798, @7:0.2960, @8:0.3096, @9:0.3210, @10:0.3314
+
+The NDCG values are most useful to examine since this allows us to compare accuracy across different queries.  The potential value of NDCG ranges from **0.0** to **1.0**, with 1.0 being a perfect model that exactly matches the ideal ranking.  
```

Comment:
```suggestion
The NDCG values are most useful to examine since this allows us to compare accuracy across different datasets.  The potential value of NDCG ranges from **0.0** to **1.0**, with 1.0 being a perfect model that exactly matches the ideal ranking.  
```